docs/src/misc.rst

```diff
@@ -246,6 +246,12 @@ API
 
     Frees the `cpu_infos` array previously allocated with :c:func:`uv_cpu_info`.
 
+.. c:function:: int uv_cpumask_size(void)
+
+    Returns the maximum size of the mask used for process/thread affinities.
```

Comment:
This function already exists in `master` in development toward 2.0.  It was added by #1527.  Please update the docs here to match:

```
    Returns the maximum size of the mask used for process/thread affinities,
    or ``UV_ENOTSUP`` if affinities are not supported on the current platform.
```

Reply:
I'm a bit confused on how best to go about this because I'm not familiar with how the libuv project does their branch management. The changes you're citing are on the master branch, but this PR is targeting the v1.x branch. Should it be targeting master instead? If so, will the changes be backported to a 1.x or 2.x release?

Reply:
`v1.x` targets the `1.x` release series and `master` currently targets `2.x`.  This PR is for v1.x currently, so we are effectively backporting `uv_cpumask_size` from 2.x development to 1.x.  We should make it consistent with what will eventually be released as 2.x.

Reply:
Would it make more sense for this to target 2.x instead?

Reply:
I'll defer to libuv maintainers for that decision.

The reason *process* affinity changes (which I contributed) are targeted to 2.x is because they require an ABI change in the process options structure.

Reply:
@cjihrig, @santigimeno: Thoughts on the target branch here?

Reply:
I think the changes here could target v1.x (unless there is some ABI breakage I'm missing). However, in that case we might want to backport https://github.com/libuv/libuv/pull/1527/commits/6feeb99d22a4c01d686667cf3b70d3933b67cd8f from master to v1.x to avoid duplicated work, and keep the contribution in @bradking's name.

Reply:
Okay. There appears to be a few things in the master branch that would be beneficial here, though there doesn't appear (at least to me) to be any technical reason not target 1.x. However, Julia recently began targeting libuv 2.x rather than 1.x from our fork so from our perspective it might be cleaner to target master and have the changes backported to 1.x at the maintainers' discretion.

Reply:
@ararslan I think if from your point of view is easier targeting master, please do it. If someone later feels strongly about this functionality to be in `v1.x`, it can be done then.